### PR TITLE
Grey Wolf Optimization Added

### PR DESCRIPTION
**Pull request by rayarka26**
Issue Number 1008

## 🛠️ Issue (Number)

Issue no #1008 

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots

### CODE REVIEW

**Comment by rayarka26** [comment]
The Correct output for a particular objective function screenshot is attached
It contains C, C++ and Python Codes
Screenshot (66)

**Review by 072arushi — APPROVED** [review]

**Review by 072arushi — APPROVED** [review]

**Review by Arun9739 — APPROVED** [review]

File: Python/Optimization/tempCodeRunnerFile.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (Kumar-laxmi) [inline Python/Optimization/tempCodeRunnerFile.py:1]: Remove this file

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Please add the JAVA Program file

**Comment by rayarka26** [comment]
@Kumar-laxmi java code has been uploaded and all changes done.
Output remains same

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Place all java class codes in one file

**Comment by rayarka26** [comment]
@Kumar-laxmi done please check

**Comment by rayarka26** [comment]
@Kumar-laxmi i also wanted to contribute one more algorithm  as issue number #958 , which you missed please check once

**Review by Kumar-laxmi — APPROVED** [review]

**Pull request merged**